packages/scene-composer/src/__snapshots__/SceneViewer.spec.tsx.snap

```diff
@@ -1,25 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`SceneViewer should render correctly 1`] = `
-.c0 {
-  position: absolute;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-}
-
-.c0 canvas {
-  outline: none;
-  -webkit-tap-highlight-color: rgba(255,255,255,0);
-}
-
 <div
-  className="c0 sceneViewer"
+  className="sc-cWSHoV icqfWN sceneViewer"
```

Comment:
Is this going to lead to inconsistent failures? The random classnames have caused issues in the past

Reply:
I don't think it would cause any more issues than it did before as the previous class names were random.

Reply:
It looks more deterministic now. Classes are named `c0`, then children are `c1`, etc.

Reply:
Are we opposed to trying it & seeing if it works before removing functionality that is designed to make us more light weight? I'd really like to only turn this off if it is determined to be a problem, especially since I think it's much more readable.

Reply:
That's fine, I just want to make sure we have some confidence and testing done to see that it won't immediately cause failures in the future